hazelcast/src/main/java/com/hazelcast/jet/impl/LightMasterContext.java

```diff
@@ -70,44 +71,45 @@ public String toString() {
     };
 
     private final NodeEngine nodeEngine;
-    private final DAG dag;
     private final long jobId;
 
     private final ILogger logger;
     private final String jobIdString;
+    private final long startTime = System.nanoTime();
 
-    private Map<MemberInfo, ExecutionPlan> executionPlanMap;
+    private final Map<MemberInfo, ExecutionPlan> executionPlanMap;
     private final AtomicBoolean invocationsCancelled = new AtomicBoolean();
     private final CompletableFuture<Void> jobCompletionFuture = new CompletableFuture<>();
-    private Set<Vertex> vertices;
+    private final Set<Vertex> vertices;
 
+    @SuppressWarnings("checkstyle:ExecutableStatementCount")
     public LightMasterContext(NodeEngine nodeEngine, DAG dag, long jobId) {
```

Comment:
How about naming it `LightJobCoordinatorContext`? I think master term for this does not properly fit.

Reply:
You're right, we use two terms: master member and coordinator member. For normal jobs, the master is always the coordinator, but not for light jobs.

In that case i'd also rename `MasterContext` and that will be a change on a lot of lines, so let's not do it here.